test/nodes/test_router_llm.py

```
@@ -0,0 +1,604 @@
+
+    def test_unknown_model_defaults_to_tier2(self):
+        info = _get_model_info('some-custom-model')
+        assert info['tier'] == 2
+        assert info['provider'] == 'unknown'
+        assert info['model'] == 'some-custom-model'
+
+    def test_model_tiers_has_nine_entries(self):
+        assert len(MODEL_TIERS) == 9
+
+    def test_each_entry_has_required_keys(self):
+        for name, entry in MODEL_TIERS.items():
```

Ruff (0.15.7)

[warning] Loop control variable `name` not used within loop body

Rename unused `name` to `_name`

(B007)